src/ORM/Association.php

```diff
@@ -1279,6 +1289,22 @@ protected function _extractFinder($finderData)
     }
 
     /**
+     * Gets the table class name.
+     *
+     * @param string $alias The alias name you want to get.
+     * @param array $options Table options array.
+     * @return string
+     */
+    protected function _getClassName($alias, array $options = [])
+    {
+        if (empty($options['className'])) {
+            $options['className'] = Inflector::camelize($alias);
```

Comment:
@lorenzo i's a copy-paste from `TableLocator`. Probably to make sure that `my_categories` alias would convert nicely to `App\Table\MyCategoriesTable`.